web/views.py

```diff
@@ -841,3 +841,20 @@ def get_context_data(self, **kwargs):
         ]
 
         return context
+
+
+class FaqView(generic.TemplateView):
+    template_name = "web/faq.html"
+
+    def get_context_data(self, **kwargs):
+        context = super().get_context_data(**kwargs)
+        if get_language() == "ja":
```

Comment:
@okaycj did you see the sentry error for this line? `get_language` is undefined. I think we need to edit the import line for translation to include `get_language`:

```
from django.utils.translation import get_language, gettext_lazy as _
```

Reply:
Woof!  I don't like this, so I changed our linter to flag this as an issue.  This change is in PR https://github.com/lookit/lookit-api/pull/1397.

Reply:
I know!! 😬 I also don't get how the import line change wasn't included in the PR (I'm assuming it was present in the contributor's code but haven't checked). 
Anyway, thanks for setting up the linter to catch this.